src/include/icastats.h

```diff
@@ -167,5 +212,26 @@ char *get_next_usr();
 void stats_reset();
 int delete_all();
 
+static inline int aes_directed_fc_stats_ofs(unsigned int fc)
```

Comment:
Did you check that this function actually gets inlined?

Reply:
No, but even if not, does it really matter? Its still quite quick....